### PR TITLE
Make scala libraries not build against abi jars

### PR DESCRIPTION
**Pull request by kageiit**
Similar to kotlin

### CODE REVIEW

**Comment by kageiit** [comment]
cc @jkeljo seems like kotlin has the similar issues to kotlin

File: src/com/facebook/buck/jvm/scala/ScalaLibraryBuilder.java [hunk 1]
```diff
@@ -51,6 +51,7 @@
         cellRoots,
         javaBuckConfig);
     this.scalaBuckConfig = scalaBuckConfig;
+    setCompileAgainstAbis(false);
```
Comment (ttsugriy) [inline src/com/facebook/buck/jvm/scala/ScalaLibraryBuilder.java]: please add a comment explaining why this is necessary

Comment (kageiit) [reply src/com/facebook/buck/jvm/scala/ScalaLibraryBuilder.java]: Added a comment to point to https://github.com/facebook/buck/issues/1386

**Comment by jkeljo** [comment]
Not terribly surprising. Thanks for the PR, we'll get it in.

**Pull request closed without merging**